generator/feature_segments_checker/feature_segments_checker.cpp

```diff
@@ -0,0 +1,220 @@
+#include "generator/srtm_parser.hpp"
+
+#include "routing/bicycle_model.hpp"
+
+#include "coding/file_name_utils.hpp"
+
+#include "geometry/mercator.hpp"
+#include "geometry/point2d.hpp"
+
+#include "indexer/classificator_loader.hpp"
+#include "indexer/feature.hpp"
+#include "indexer/feature_processor.hpp"
+#include "indexer/map_style_reader.hpp"
+
+#include "platform/platform.hpp"
+
+#include "base/logging.hpp"
+
+#include "3party/gflags/src/gflags/gflags.h"
+
+#include "std/iostream.hpp"
+#include "std/map.hpp"
+#include "std/string.hpp"
+
+DEFINE_string(srtm_path, "", "Path to directory with SRTM files");
+DEFINE_string(mwm_path, "", "Path to an mwm file.");
+
+namespace
+{
+routing::BicycleModel const & GetBicycleModel()
+{
+  static routing::BicycleModel const instance;
+  return instance;
+}
+
+template <typename K, typename V>
+void IncrementStats(K const & k, map<K, V> & m)
+{
+  if (m.find(k) == m.cend())
+    m[k] = 0;
+  m[k] += 1;
+}
+
+template <typename Cont>
+void PrintCont(Cont const & cont, string const & title, string const & msgText1,
```

Comment:
I'm sure it makes this method more understandable. The method is quite easy and reading the comment could take even more time then reading the method.